From e002ef547a04f29ac4a0c3a614695233964c9077 Mon Sep 17 00:00:00 2001
From: Deepali Chourasia <deepch23@in.ibm.com>
Date: Thu, 11 Jan 2024 11:01:53 +0000
Subject: [PATCH] Remove bazel shutdown call from jax code

---
 build/build.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/build.py b/build/build.py
index 41994c4b4..a05d7d0cd 100755
--- a/build/build.py
+++ b/build/build.py
@@ -595,7 +595,7 @@ def main():
     print(" ".join(build_pjrt_plugin_command))
     shell(build_pjrt_plugin_command)
 
-  shell([bazel_path] + args.bazel_startup_options + ["shutdown"])
+ #shell([bazel_path] + args.bazel_startup_options + ["shutdown"])
 
 
 if __name__ == "__main__":
-- 
2.40.1

